Solutions/Windows Security Events/Hunting Queries/MultipleExplicitCredentialUsage4648Events.yaml

```diff
@@ -31,13 +31,13 @@ query: |
   | where SubjectUserSid != 'S-1-0-0' // this is the Nobody SID which really means No security principal was included.
   | where not(SubjectUserSid matches regex WellKnownLocalSIDs) //excluding system account/service account as this is generally normal
   | where TargetInfo has '/' //looking for only items that indicate an interesting protocol is included
-  | where Computer !has tostring(split(TargetServerName,'$')[0])
-  | where TargetAccount !~ tostring(split(SubjectAccount,'$')[0])
-  | extend TargetInfoProtocol = tolower(split(TargetInfo, '/')[0]), TargetInfoMachine = toupper(split(TargetInfo, '/')[1])
+  | where Computer !has tostring(split(TargetServerName,'$', 0)[0])
```

Comment:
Sure @petebryan. It was advised to use split function's index over subscript, but also wanted to return string rather than an array, so ended up using both.